src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy

```diff
@@ -0,0 +1,192 @@
+package org.gradle.android
+
+class SimpleAndroidApp {
+    final File projectDir
+    private final File cacheDir
+    final String androidVersion
+    private final boolean reverseApply
+
+    SimpleAndroidApp(File projectDir, File cacheDir, String androidVersion, boolean reverseApply = false) {
+        this.reverseApply = reverseApply
+        this.projectDir = projectDir
+        this.cacheDir = cacheDir
+        this.androidVersion = androidVersion
+    }
+
+    def writeProject() {
+        def app = 'app'
+        def appPackage = 'org.gradle.android.example.app'
+        def appActivity = 'AppActivity'
+
+        def library = 'library'
+        def libPackage = 'org.gradle.android.example.library'
+        def libraryActivity = 'LibraryActivity'
+
+        file("settings.gradle") << """
```

Comment:
Do you think it would be a good idea to remove the cache configuration from the `SimpleAndroidApp`? It could go better into its own cache fixture. We don't have to though, now.

Reply:
Sure. But what we should do even better is to add this feature to TestKit, no?

Reply:
That would be great! Do we have a card already?

Reply:
Now we do: https://github.com/gradle/build-cache/issues/890